plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any, Dict, List, Optional
+
+from hydra.core.config_store import ConfigStore
+from hydra.types import ObjectConf
+from omegaconf import MISSING
+
+
+@dataclass
+class RayInitConf(Dict[str, Any]):
+    """ ray.init(**ray_init_config) """
+
+    address: Optional[str] = MISSING
+    num_cpus: Optional[int] = MISSING
+    num_gpus: Optional[int] = MISSING
+
+
+@dataclass
+class RayRemoteConf(Dict[str, Any]):
+    """ @ray.remote(**ray_remote_cfg)"""
+
+    num_cpus: Optional[int] = MISSING
+    num_gpus: Optional[int] = MISSING
+
+
+@dataclass
+class RayConf:
+    ray_init_cfg: RayInitConf = MISSING
+    ray_remote_cfg: RayRemoteConf = MISSING
+
+
+@dataclass
+class RayLocalConf(RayConf):
+    pass
```

Comment:
What is the purpose of this class?

Also, I think this is nicer:
```suggestion
class RayLocalConf(RayConf):
    ...
```